src/VaccinTracker/carteRepartitionVaccins.php

```diff
@@ -158,160 +157,87 @@ function buildBarChart(){
         * Sans quoi chaqué département commence son clignotement au moment où on lui attribue
         * la classe selected.
         */
-        var stopAnimation = function (){
+        var stopAnimation = function () {
             $("#map").removeClass("animated")
         }
 
-        var startAnimation = function (){
+        var startAnimation = function () {
             $("#map").addClass("animated")
         }
 
-        // fetch('https://raw.githubusercontent.com/rozierguillaume/covid-19/master/data/france/stats/reffectif_region.json')
-        //     .then(response => {
-        //         if (!response.ok) {
-        //             throw new Error("HTTP error " + response.status);
-        //         }
-        //         return response.json();
-        //     })
-        //     .then(json => {
-        //         reffectifRegions = json;
-        //     });
-
-        // fetch('https://raw.githubusercontent.com/rozierguillaume/covid-19/master/data/france/stats/saturation_rea_regions.json')
-        //     .then(response => {
-        //         if (!response.ok) {
-        //             throw new Error("HTTP error " + response.status);
-        //         }
-        //         return response.json();
-        //     })
-        //     .then(json => {
-        //         saturationReaRegions = json;
-        //     });
-
-
         function afficherRegion(nomRegion, numeroRegion) {
-
-            vaccinesRegion = donneesRegions[nomRegion]["vaccines"]
-            vaccinesRegionPop = (vaccinesRegion/donneesRegions[nomRegion]["population"]*100).toFixed(3)
-
-            dateMaj = dateMaj
-
-            if (vaccinesRegion<=0){
-                vaccinesRegion = "--"
+            vaccinesRegion = donneesRegions[nomRegion]["vaccines"];
+            vaccinesRegionPop = (vaccinesRegion / donneesRegions[nomRegion]["population"] * 100).toFixed(2);
+            if (vaccinesRegion <= 0) {
+                vaccinesRegion = "--";
             }
-
             if ($('#' + numeroRegion).length > 0) {
                 return;
             }
             content = $('#regionTemplate').html();
             content = content.replace(/nomRegion/g, nomRegion);
             content = content.replace(/numeroRegion/g, numeroRegion);
-            content = content.replace(/vaccinesRegion/g, vaccinesRegion);
+            content = content.replace(/vaccinesRegion/g, vaccinesRegion.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1 "));
```

Comment:
Si j'ai bien compris la regexp, on pourrait la remplacer par l'utilisation de [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString).

```suggestion
            content = content.replace(/vaccinesRegion/g, vaccinesRegion.toLocaleString());
```

Reply:
Top merci.... Je connaissais pas cette méthode!